client/containers/DashboardEdges/dashboardEdges.scss

```diff
@@ -0,0 +1,26 @@
+.dashboard-edges-container {
+    .dashboard-edges-listing-component {
+        margin-top: 24px;
+        overflow-y: unset;
+    }
+    .dashboard-edge-listing-item {
+        display: flex;
+        &.is-dragging {
+            background: white;
+        }
+        .pub-edge-component {
+            transition: 100ms opacity ease-in-out;
+        }
+        &.not-approved {
+            .pub-edge-component {
+                opacity: 0.5;
+            }
+        }
```

Comment:
Can we organize the selectors here, maybe by putting the states (`is-` and `-not`) together at the top?